frontend/src/container/ListAlertRules/ListAlert.tsx

```diff
@@ -142,6 +179,10 @@ function ListAlert({ allAlertRules, refetch }: ListAlertProps): JSX.Element {
 				return prev - next;
 			},
 			render: DateComponent,
+			sortOrder:
+				sortedInfo.columnKey === DynamicColumnsKey.CreatedAt
```

Comment:
Do we need safety checks for sortedInfo.columnKey and order ?

Reply:
No, an empty string has been passed as a column key by default.